docs/src/main/sphinx/release/release-394.md

```diff
@@ -0,0 +1,88 @@
+# Release 394 (31 August 2022)
+
+## General
+
+* Add support for `EXPLAIN (TYPE DISTRIBUTED, FORMAT JSON)`. ({issue}`12968`)
```

Comment:
I would describe this in prose to that it's clear what's new:

* Add support for JSON format to output of `EXPLAIN`